doc/releases/migration-guide-3.7.rst

```diff
@@ -126,6 +126,9 @@ Display
 Flash
 =====
 
+* The STM32 devices with xSPI peripherals, like the stm32h5 serie, now have their own compatible
+  and flash controller driver
+
```

Comment:
Not up to date. Also, for now we're keeping compatibility with ospi, so this can be removed.

Reply:
> Not up to date. Also, for now we're keeping compatibility with ospi, so this can be removed.

OK, let's reserve this for the next stm32 xspi driver update